src/test/groovy/com/lesfurets/jenkins/unit/declarative/TestDeclarativePipeline.groovy

```diff
@@ -639,6 +639,13 @@ class TestDeclarativePipeline extends DeclarativePipelineTest {
         assertJobStatusSuccess()
     }
 
+    @Test void should_kubernetes_default_agent() throws Exception {
+        runScript('Kubernetes_Default_Jenkinsfile')
+        printCallStack()
+        assertCallStack().contains('Executing on agent [kubernetes')
```

Comment:
Is this a typo? Where is the other `]`? What does the callstack line actually look like?

Reply:
it has a bunch of other information which didn't seem entirely relevant. Here's the whole thing, I'm happy to write the assertion however you'd prefer: 

```
com.lesfurets.jenkins.unit.declarative.TestDeclarativePipeline > should_kubernetes_default_agent STANDARD_OUT
    Executing on agent [kubernetes:[activeDeadlineSeconds:0, instanceCap:0, stages:[:], slaveConnectTimeout:0, idleMinutes:0]]
    Hello, JDK

       Kubernetes_Default_Jenkinsfile.run()
          Kubernetes_Default_Jenkinsfile.pipeline(groovy.lang.Closure)
             Kubernetes_Default_Jenkinsfile.echo(Executing on agent [kubernetes:[activeDeadlineSeconds:0, instanceCap:0, stages:[:], slaveConnectTimeout:0, idleMinutes:0]])
             Kubernetes_Default_Jenkinsfile.stage(Example Test, groovy.lang.Closure)
                Kubernetes_Default_Jenkinsfile.echo(Hello, JDK)
                Kubernetes_Default_Jenkinsfile.sh(java -version)
```

Reply:
Thanks for the explanation, I think we can leave the test as is.